### PR TITLE
bpo-11122: always use rpmbuild in bdist_rpm

### PR DESCRIPTION
**Pull request by n0npax**
https://bugs.python.org/issue11122

### CODE REVIEW

File: Lib/distutils/command/bdist_rpm.py [hunk 1]
```diff
@@ -309,10 +309,7 @@ def run(self):
 
         # build package
         log.info("building RPMs")
-        rpm_cmd = ['rpm']
-        if os.path.exists('/usr/bin/rpmbuild') or \
-           os.path.exists('/bin/rpmbuild'):
-            rpm_cmd = ['rpmbuild']
+        rpm_cmd = ['rpmbuild']
```
Comment (merwok) [inline Lib/distutils/command/bdist_rpm.py:312]: Should line 338 be changed too?

**Comment by csabella** [comment]
In #10172, the suggested change was to use `if shutil.which('rpmbuild')`, but that also added an additional import.  I closed that pull request as a duplicate of this one.

**Review by ericvsmith — APPROVED** [review]
I don't see much point in running `which`, just to find the path. It doesn't buy you anything vs. just running the command.

I think the patch as-is is fine.

**Comment by merwok** [comment]
There was no answer to my review question!

**Pull request merged**